recipe/meta.yaml

```
@@ -26,30 +24,27 @@ build:
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
-  build:
-    - patch # [unix]
-    - m2-patch # [win]
   host:
     - python
     - pip
     - hatchling >=1.10.0
-    - incremental >=22.10.0 
+    - hatch-fancy-pypi-readme >=22.5.0
+    - incremental >=24.7.0 
   run:
     - python
-    - attrs >=21.3.0
-    - automat >=0.8.0
+    - zope.interface >=5
     - constantly >=15.1
+    - incremental >=24.7.0
+    - automat >=24.8.0
     - hyperlink >=17.1.1
-    - incremental >=22.10.0
+    - attrs >=22.2.0
     - typing_extensions >=4.2.0
-    - twisted-iocpsupport >=1.0.2,<2  # [win]
-    - zope.interface >=5
+    - twisted-iocpsupport >=1.0.2  # [win]
     - pywin32  # [win]
```

```suggestion
    - pywin32  !=226  # [win]
```

https://github.com/twisted/twisted/blob/twisted-24.11.0/pyproject.toml#L117